rubocop-discourse.gemspec

```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |s|
   s.name = "rubocop-discourse"
-  s.version = "3.8.1"
+  s.version = "4.0.0"
```

Comment:
Bumped major version since this is a backwards incompatible change.

Reply:
In lint configs pretty much every change can be "breaking" (i.e. require manual or automated changes to the code) but I'd bump major version only if it requires changes to the linting setup itself, like if you have to update your gemfile/rubocop config file/etc. We've been adding new rules in minor versions.

Reply:
I see. I just assumed we were following semver here.